src/ccalls.jl

```diff
@@ -15,6 +15,35 @@ function EvalStringEx(cmd::String)
     return res
 end
 
+"""
+    EvalString(cmd::String)
+
+Let GAP execute the command(s) given by `cmd` and returns the last result.
+An error is thrown if no value is returned.
+
+# Examples
+```jldoctest
+julia> GAP.EvalString( "1+2" )
+3
+
+julia> GAP.EvalString( "x:= []; y:= 0" )
+0
+
+julia> GAP.EvalString( "Add( x, 1 )" )
+ERROR: Error thrown by GAP: Error, List Element: <list>[2] must have an assigned value
```

Comment:
See also #474 (this is not a request to change this PR, I just wanted to point out the issues it motivated me to submit ;-)